README.md

```diff
@@ -45,6 +45,10 @@ sudo mv cmctl /usr/local/bin
 # or `sudo mv cmctl /usr/local/bin/kubectl-cert_manager` to use `kubectl cert-manager` instead.
 ```
 
+### Distributed releases
+
+[![Packaging status](https://repology.org/badge/vertical-allrepos/cmctl.svg)](https://repology.org/project/cmctl/versions)
+
```

Comment:
Maybe add a small sentence about where this is coming from and potentially a hint to how to get updates.